ios/TabViewImpl.swift

```diff
@@ -188,9 +188,11 @@ extension View {
   
   @ViewBuilder
   func tabBadge(_ data: String?) -> some View {
-    if #available(iOS 15.0, macOS 15.0, visionOS 2.0, *) {
+    if #available(iOS 15.0, macOS 15.0, visionOS 2.0, tvOS 15.0, *) {
       if let data = data, !data.isEmpty {
+        #if !os(tvOS)
         self.badge(data)
+        #endif
```

Comment:
We need to return `self` either way, this can cause another modifier not to be applied. 

```suggestion
        #if !os(tvOS)
        self.badge(data)
        #else
        self
        #endif
```

Reply:
```suggestion
        #if !os(tvOS)
        self.badge(data)
        #else
        self
        #endif
```